src/README.md

```diff
@@ -14,22 +14,23 @@ This documentation is written for different audiences and is useful for develope
 
 ## How To Read This Documentation
 
-In general, it is assumed that a reader has basic understanding of traditional domains and how blockchain works. Some developer topics will require familiarity with Solidity language. Sections that require knowledge of specific concepts contain links to resources and prerequisites, including both external ones and references to our own documentation.
+In general, it is assumed that a reader has a basic understanding of traditional domains and how blockchain works. Some developer topics will require familiarity with Solidity language. Sections that require knowledge of specific concepts contain links to resources and prerequisites, including both external ones and references to our documentation.
 
-Developer guides contain examples of code, mostly written in JavaScript and Solidity. The choice of the JavaScript language was motivated by the fact that the most mature libraries of our own ecosystem are built with JavaScript and we assume that this language is available for the majority of our technical readers, so they can get the examples working without much effort.
+Developer guides contain examples of code, mostly written in JavaScript and Solidity. The choice of the JavaScript language was motivated by the fact that the most mature libraries of our ecosystem are built with JavaScript and we assume that this language is available for the majority of our technical readers, so they can get the examples working without much effort.
 
 ### Navigating
 
-- If you want an understanding of how **Unstoppable Domains work, its architecture and capabilities**, check out [General Overview](domain-registry-essentials/architecture-overview.md).
-- If you're a **developer** interested in **resolving records for cryptocurrency payments**, check out [Resolving Domain Records](domain-registry-essentials/resolving-domain-records.md).
-- If you're a **developer** interested in **how to resolve our domains in browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
+* If you want an understanding of how **Unstoppable Domains work, its architecture and capabilities**, check out [General Overview](domain-registry-essentials/architecture-overview.md).
+* If you're a **developer** interested in **resolving records for cryptocurrency payments**, check out [Resolving Domain Records](domain-registry-essentials/resolving-domain-records.md).
+* If you're a **developer** interested in **how to resolve our domains in the browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
```

Comment:
```suggestion
* If you're a **developer** interested in **how to resolve our domains in a browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
```

Reply:
What's meant by "fix every other article before"? Switching "a" for "the" shouldn't need any other changes